packages/deployment/ansible/roles/cosmos-genesis/files/faucet-helper.sh

```diff
@@ -29,9 +29,13 @@ show-faucet-address)
 esac
 
 networkName=$(basename "$thisdir")
-netconfig=$(curl "https://$networkName.agoric.net/network-config")
-chainName=$(echo "$netconfig" | jq -r .chainName)
-read -r -a origRpcAddrs <<<"$(echo "$netconfig" | jq -r .rpcAddrs[])"
+if netconfig=$(curl "https://$networkName.agoric.net/network-config"); then
+  chainName=$(echo "$netconfig" | jq -r .chainName)
+  read -r -a origRpcAddrs <<<"$(echo "$netconfig" | jq -r .rpcAddrs[])"
+else
+  chainName=$(cat "$thisdir/ag-chain-cosmos/chain-name.txt")
```

Comment:
I was considering doing `ag-setup-cosmos show-config` instead to keep the alternatives consistent, but this roughly does the same thing